pyproject.toml

```diff
@@ -0,0 +1,43 @@
+[build-system]
+requires = ["setuptools>=61.2"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "ppafm"
+dynamic = ["version"]
+description = "Classical forcefiled model for simulation atomic force microscopy images."
```

Comment:
Let's fix the bad English here. I would change that to "Classical force field model for simulating atomic force microscopy images."

I would also change the description on the main page, but I don't think I have the rights to do that.

Reply:
done